src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -198,6 +198,13 @@ CHIP_ERROR CopyTextRecordValue(char * buffer, size_t bufferLen, CommissioningMod
     return CopyTextRecordValue(buffer, bufferLen, static_cast<uint16_t>(value));
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+CHIP_ERROR CopyTextRecordValue(char * buffer, size_t bufferLen, JointFabricMode value)
```

Comment:
Again, BitFlags to represent the flags, and Raw() to get an integer.

Reply:
Addressed in the latest commit. Thanks for your feedback.